zcash_client_backend/src/data_api/wallet.rs

```diff
@@ -207,7 +207,7 @@ pub fn create_spend_to_address<DbT, ParamsT>(
     min_confirmations: NonZeroU32,
     change_memo: Option<MemoBytes>,
 ) -> Result<
-    TxId,
+    Vec<TxId>,
```

Comment:
Update the documentation for this function to not assume a single transaction is created.